.gitignore

```diff
@@ -1,12 +1,12 @@
 .DS_Store
 .idea
-suite/repo
 firmware/repo
 trezord-go/repo
+trezord-go/bin/*
```

Comment:
We do not want to ignore that no? I mean the binaries are baked in via git intentionally.

Reply:
Yes, that makes sense, I am deleting that, it was sort of an experiment.

What feels strange is that I have run the download.sh, it updated all 4 of those bridge binaries, and they all showed in git changes - like this line was not having any effect. Before running the download.sh, these bridge were there, but not working for me (nixos was complaining about file not being there)

Reply:
Solved via #66.